code/_core/mob/living/simple/goblin_king.dm

```diff
@@ -0,0 +1,115 @@
+/mob/living/simple/goblin_king
+	name = "goblin king"
+	boss_icon_state = "goblin_king"
+	icon = 'icons/mob/living/simple/goblin_king.dmi'
+	icon_state = "living"
+	damage_type = /damagetype/unarmed/claw/
+	class = /class/goblin_king/
+	ai = /ai/boss/goblin_king/
+
+	boss_loot = /loot/lavaland/goblin_king
+
+	value = 4000
+
+	health_base = 15000
+	stamina_base = 4000
+	mana_base = 1000
+
+	move_delay = BOSS_TICK*3
+
+	stun_angle = 0
+
+	attack_range = 2
+
+	force_spawn = TRUE
+	boss = TRUE
+
+	armor_base = list(
+		BLADE = AP_GREATSWORD,
+		BLUNT = AP_GREATSWORD,
+		LASER = AP_GREATSWORD,
+		HEAT = AP_GREATSWORD,
+		COLD = AP_GREATSWORD,
+		HOLY = -AP_GREATSWORD,
+		DARK = AP_GREATSWORD,
+		FATIGUE = AP_GREATSWORD,
+		ION = INFINITY,
+		PAIN = INFINITY
+	)
+
+	status_immune = list(
+		STUN = TRUE,
+		SLEEP = TRUE,
+		PARALYZE = TRUE,
+		STAMCRIT = TRUE,
+		STAGGER = TRUE,
+		CONFUSED = TRUE,
+		CRIT = TRUE,
+		REST = TRUE,
+		ADRENALINE = TRUE,
+		DISARM = TRUE,
+		DRUGGY = TRUE,
+		FIRE = FALSE
+	)
+
+	iff_tag = "Goblin"
+	loyalty_tag = "Goblin"
+
+	fatigue_from_block_mul = 0
+
+	size = SIZE_BOSS
+
+	sprint_delay_mul = 1
+	jog_delay_mul = 3
+	walk_delay_mul = 4
+
+	enable_medical_hud = FALSE
+	enable_security_hud = FALSE
+
+	blood_volume = 1000
```

Comment:
For a 15000 hp boss, I think blood volume should be about 5000 as they'd die from blood loss before they die from health loss.